### DISCUSSION TITLE
Blueman connection disconnection pop up window

### DISCUSSION THREAD

**Discussion by MacWingman** [blueman-project/blueman#2470] [Q&A]
blueman:
BlueZ:
Distribution: Raspberry Pi Bookworm
Desktop environment:

-->I have a QT application that I have ported from multiple versions of the Raspberry Pi OS.  Other than the pairing popup, connections and disconnections from Bluetooth devices did not raise any popup notification windows.  The application recognized connections and disconnections and notified the user appropriately.  

Now with Bookworm, there is a popup window notification every time there is a connection or disconnection of either of the two bluetooth devices that the application interfaces with.  This popup does not time out and go away, but requires a physical acknowledgement.  This is not desirable and is in fact completely undesirable in the context of this application.  Never was a problem before, and I don't understand the need to add this non-feature now.  It should be at least configurable!  

How do I eliminate this popup?  If not currently configurable, it should be made configurable, or discarded.

**Comment by infirit** [comment]
If you see windows it means blueman is in the fallback path when it can't find a proper notification daemon.
